packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js

```diff
@@ -55,7 +54,28 @@ const calculateDomain = (series, data, axesDomains, argumentAxisName) =>
     axesDomains,
   );
 
+const zoomingDomains = (axes, calculatedDomains) => axes.reduce(
```

Comment:
I think "zoomingDomains" is not a good name for this function.

Reply:
Maybe 'adjustDomains'?

Reply:
good